File: Functions/4.py
def fact(a):
    p = 1
    for i in range(1,a+1):
        p*=i
    return p

def calculate_nCr(n,r):
    result = (fact(n))/(fact(r)*fact(n-r))
    return result

print(calculate_nCr(8,2))
print(calculate_nCr(6,3))
print(calculate_nCr(8,7))

File: Functions/4.cpp
#include<bits/stdc++.h>
using namespace std;

int fact(int a){
    int fact = 1;
    for(int i=1;i<a+1;i++){
        fact*=i;
    } 
    return fact;
}

float nCr(int n, int r){
    float result = (fact(n))/(fact(r)*fact(n-r));
    return result;
}

int main(){
    int n,r;
    cout<<"Enter n: ";
    cin>>n;
    cout<<"Enter r: ";
    cin>>r;
    cout<<nCr(n,r)<<endl;
}